app/models/product.rb

```diff
@@ -0,0 +1,2 @@
+class Product < ActiveRecord::Base
+end
```

Comment:
I would like you to add validation for :name and :price

Reply:
Check!